types/got/index.d.ts

```diff
@@ -113,16 +115,20 @@ declare namespace got {
     }
 
     interface GotOptions<E extends string | null> extends InternalRequestOptions {
+        baseUrl?: string;
+        headers?: { [key: string]: string };
```

Comment:
Property `headers` will be inherited from `https.RequestOptions`.

Reply:
Good catch! I will remove that line.